src/routes/publish.ts

```diff
@@ -64,6 +71,19 @@ publish.post('/:datasetId/sources', fetchDataset, upload.none(), sources);
 /* Tasklist */
 publish.get('/:datasetId/tasklist', fetchDataset, taskList);
 
+/* Cube Preview */
+publish.get('/:datasetId/cube-preview', fetchDataset, cubePreview);
+publish.get('/:datasetId/cube-preview/csv', fetchDataset, downloadAsCSV);
+publish.get('/:datasetId/cube-preview/parquet', fetchDataset, downloadAsParquet);
+publish.get('/:datasetId/cube-preview/excel', fetchDataset, downloadAsExcel);
+publish.get('/:datasetId/cube-preview/cube', fetchDataset, downloadAsDuckDb);
```

Comment:
As mentioned in the controllers, this could probably just be 

```
publish.get('/:datasetId/cube-preview', fetchDataset, cubePreview);
```

with URLs like `/:datasetId/cube-preview?format=csv` etc

Reply:
Save for a future refactor.